engine/src/engine.rs

```diff
@@ -1136,12 +1141,14 @@ pub fn set_balance<I: IO>(io: &mut I, address: &Address, balance: &Wei) {
 }
 
 pub fn remove_balance<I: IO + Copy>(io: &mut I, address: &Address) {
-    let balance = get_balance(io, address);
+    // The `unwrap` is safe here because if the connector
+    // is implemented correctly then the "Eth on Aurora" wll never underflow.
+    let balance = get_balance(io, address).try_into_u128().unwrap();
```

Comment:
I think a better way to phrase this justification would be something like

"All ETH on Aurora originates from transfers over the bridge into the NEP-141 contract on NEAR. In particular, all ETH in the EVM will be held by the `aurora` account in that NEP-141 contract. Since the NEP-141 standard represents balances as u128, it means there can never be more than u128::MAX total ETH in the EVM (if someone were to try, the NEAR transaction would fail). Therefore there can never be an EVM address with more than u128::MAX either."

But I agree, the unwrap here is safe in practice.